vere/sist.c

```diff
@@ -779,11 +660,7 @@ _sist_rest()
       u3_lo_bail();
     }
 
-    if ( u3r_mug('f') == led_u.mag_l ) {
-      _sist_rest_nuu(&u3Z->lug_u, led_u, ful_c);
-      fid_i = u3Z->lug_u.fid_i;
-    }
-    else if (u3r_mug('g') != led_u.mag_l ) {
+    if (u3r_mug('h') != led_u.mag_l ) {
```

Comment:
It'd be nice to have an explicit case of `u3r_mug('g')` with a friendly message explaining the need to use an older interpreter with this event log (and maybe event specifying the v0.7.0 tag of this repository).

Reply:
This PR now includes `_sist_rest_nuu()`, which transforms 'g'-format event logs to 'h'-format event logs. It prints a message when it starts the conversion.